mandatory/2-function-creation.js

```diff
@@ -45,7 +67,14 @@ function canRegister(age) {}
   )
 */
 
-function countReverse(number) {}
+function countReverse(number) {
+  let reversedCount = number;
+  while (reversedCount > 0) {
+    console.log(reversedCount);
+    reversedCount--;
+    
+  }
+}
 
 /* ======= TESTS - DO NOT MODIFY ===== */
 
```

Comment:
Amazing work Alena, I really like your use of ternary operators, you really seems to understand the concept!
Everything looks perfect to me 👍 :)